### ISSUE TITLE
Benchmarking

### ISSUE THREAD

**Issue by TomLucidor** [NVlabs/Sana#5]
After taking a look at benchmarks doing comparisons, wondering if there are ways to show that Sana is really flexible or not
- Flux https://blackforestlabs.ai/announcing-black-forest-labs/
- Playground v3 https://arxiv.org/html/2409.10695v2
- SD3 https://stability.ai/news/stable-diffusion-3-research-paper
- PixArt Sigma https://arxiv.org/html/2403.04692v2
- Open arena to compare model output https://artificialanalysis.ai/text-to-image/arena

**Comment by TomLucidor** [comment]
Yeah, comparison or leaderboard or something that demo their performance vs your SOTA (preferably against SDXL tweaks, Flux, and AuraFlow WIPs)

**Comment by lawrence-cj** [comment]
Do you know how to get performance ranking from here: https://artificialanalysis.ai/text-to-image/arena?

**Comment by TomLucidor** [comment]
@lawrence-cj you mean the user preference platform? Yeah their contact form is open to accepting new models for testing so would like to see how things will pan out
